testhelpers/org.eclipse.n4js.tests.helper/src/org/eclipse/n4js/xpect/validation/suppression/SuppressIssuesSetup.xtend

```diff
@@ -38,7 +38,7 @@ import org.xpect.xtext.lib.tests.ValidationTestModuleSetup.TestingResourceValida
  *
  * To integrate this setup with an xpect runner, import this class via @XpectImport.
  *
- * For further configuration you can use {@link IssueConfiguration} in the XPECT_SETUP of specific files.
+ * For further configuration you can use {@link IssueConfiguration} in the XPECTSETUP of specific files.
```

Comment:
This was necessary, since Xpect was interpreting this whole file as a Xpect test file.